station/app/schemas/local_trains.py

```diff
@@ -27,25 +27,87 @@ class LocalTrain(DBSchema):
 
 
 class LocalTrainCreate(BaseModel):
-    train_name: str
+    train_name: Optional[str] = None
 
+class LocalTrainConfigBase(BaseModel):
+    name: str
+    image: Optional[str] = None
+    tag: Optional[str] = None
+    query: Optional[str] = None
+    entrypoint: Optional[str] = None
+    volumes: Optional[str] = None
+    env: Optional[str] = None
+    train_id: Optional[str] = None
 
-class LocalTrainAddMasterImage(BaseModel):
-    train_id: str
-    image: str
 
-class LocalTrainGetFile(BaseModel):
-    train_id: str
-    file_name: str
+class LocalTrainConfigCreate(LocalTrainConfigBase):
+    pass
+
 
-class LocalTrainAddTag(BaseModel):
+class LocalTrainConfigUpdate(LocalTrainConfigBase):
+    pass
+
+class LocalTrainAirflowConfig(BaseModel):
+    image: Optional[str] = None
+    repository: Optional[str] = None
+    tag: Optional[str] = None
+    env: Optional[str] = None
+    query: Optional[str] = None
+    entrypoint: Optional[str] = None
+    volumes: Optional[str] = None
+    train_id: Optional[str] = None
```

Comment:
I think most of the fields are already in `LocalTrainConfigBase`  can they be removed?